CodeEditModules/Modules/Keybindings/src/KeybindingManager.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  KeybindingManager.swift
+//  
+//
+//  Created by Alex on 09.05.2022.
+//
+
+import Foundation
+import SwiftUI
+
+// swiftlint:disable missing_docs
+public final class KeybindingManager {
+    public var keyboardShortcuts = [String: KeyboardShortcutWrapper]()
+
+    private init() {
+        loadKeybindings()
+    }
+
+    // swiftlint:disable missing_docs
+    public static let shared: KeybindingManager = .init()
+
+    // We need this fallback shortcut because optional shortcuts available only from 12.3, while we have target of 12.0x
+    var fallbackShortcut = KeyboardShortcutWrapper(name: "?", description: "Test", context: "Fallback",
+                                                   keybinding: "?", modifier: "shift", id: "fallback")
```

Comment:
Same thing here about method arguments. keep them on separate lines to increase readability.